qa/rolling-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -26,6 +26,9 @@
       },
       "passage_text": {
         "type": "text"
+      },
+      "passage_image": {
+        "type": "binary"
```

Comment:
type should be text in this case, we're sending that value directly to the model, so it supposed to be a base64 of the image. binary will work too, but just a bit misleading

Reply:
I have tried to simulate the same test scenario present in the opensearch documentation. Do you still want to make the change to text?

Reply:
To me we need to make it `text` and also fix the documentation. Maybe they put it there to illustrate that image can be binary. Anyway, for the purpose of this test `text` works better